tiledb/bioimg/converters/base.py

```diff
@@ -536,7 +544,10 @@ def to_tiledb(
                 original_metadata = reader.original_metadata
             else:
                 if ome_xml := reader.original_metadata.get("ome_metadata"):
-                    pruned_metadata = remove_ome_image_metadata(ome_xml)
+                    if isinstance(exclude_metadata, bool):
+                        pruned_metadata = remove_ome_image_metadata(ome_xml)
+                    else:
```

Comment:
```suggestion
                    elif callable(exclude_metadata):
```
and then raise an exception if it is neither.